### PR TITLE
Add Telemetry

### PR DESCRIPTION
**Pull request by CahidArda**
Adds telemetry headers to workflow requests.

### CODE REVIEW

File: platforms/cloudflare.ts [hunk 1]
```diff
@@ -62,10 +63,17 @@ export const serve = <TInitialPayload = unknown>(
...
     const { request, env } = getArgs(args);
-    const { handler: serveHandler } = serveBase(routeFunction, {
-      env,
-      ...options,
-    });
+    const { handler: serveHandler } = serveBase(
+      routeFunction,
+      {
+        sdk: SDK_TELEMETRY,
+        platform: "cloudflare",
```
Comment (fahreddinozcan) [inline platforms/cloudflare.ts:70]: Why don't we have runtime here?

File: platforms/express.ts [hunk 2]
```diff
@@ -43,7 +44,12 @@ export function serve<TInitialPayload = unknown>(
 
     // create handler
     const { handler: serveHandler } = serveBase<TInitialPayload>(
-      (workflowContext) => routeFunction(workflowContext),
+      routeFunction,
+      {
+        sdk: SDK_TELEMETRY,
+        platform: "express",
+        runtime: `node@${process.version}`,
```
Comment (fahreddinozcan) [inline platforms/express.ts:51]: How can we make sure it's node? It can also be bun etc... can we somehow detect that?

**Comment by fahreddinozcan** [comment]
I think we can change `platform` -> `framework`. Imo hono is not a platform etc. If we can collect if the usage is on cloudflare/vercel etc. we can write that to platform

File: platforms/express.ts (hunk 2 above)
Comment (fahreddinozcan) [reply platforms/express.ts:51]: This applies to all the configs

**Comment by fahreddinozcan** [comment]
The user should be able to opt out of telemetry logging, but it can be enabled by default

File: platforms/express.ts (hunk 2 above)
Comment (fahreddinozcan) [reply platforms/express.ts:51]: https://bun.sh/guides/util/detect-bun#:~:text=The%20recommended%20way%20to%20conditionally,being%20executed%20in%20a%20browser.

File: platforms/cloudflare.ts (hunk 1 above)
Comment (CahidArda) [reply platforms/cloudflare.ts:70]: no `process` object in cloudflare. I couldn't find an equivalent

**Review by fahreddinozcan — APPROVED** [review]

**Pull request merged**